### ISSUE TITLE
Remove animation from vishraam words changing

### ISSUE THREAD

**Issue by theamanjs** [KhalisFoundation/sttm-desktop#1490]
There is a slow animation of Vishraam Words when changing the lines, It seems disturbing. If required, an option can be given to the user if he requires animation or simple changes.

**Comment by saintsoldierx** [comment]
I disagree with this kinda. The animations are visibly nicer. We can speed up the animation but removing it to be a straight cut doesn't seem better to me. Let's not offer any option to animate it either, we don't need to go that granular.

**Comment by theamanjs** [comment]
Sure veerji, I'll speed up the animation and will give setting to disable the animation.

**Comment by saintsoldierx** [comment]
Don't give option to turn off but speed it up. We have too many options in settings and this would just be another pointless and minor setting that isn't really needed.

**Comment by theamanjs** [comment]
Got it veerji, will speed that up.
